src/token/ERC20MultiVotes.sol

```diff
@@ -66,7 +66,7 @@ abstract contract ERC20MultiVotes is ERC20, Auth {
      * @param blockNumber the block to calculate votes for.
      * @return the amount of votes.
      */
-    function getPastVotes(address account, uint256 blockNumber) public view returns (uint256) {
+    function getPastVotes(address account, uint256 blockNumber) public view virtual returns (uint256) {
```

Comment:
Why is this virtual?

Reply:
overwrote in xTRIBE for instance because not only we need a previous balance but also add normal TRIBE past votes